src/analysis/special_functions/pow.lean

```diff
@@ -933,6 +937,30 @@ begin
 end
 end limits
 
+namespace complex
+
+lemma continuous_at_cpow_zero_of_pos_re {z : ℂ} (hz : 0 < z.re) :
```

Comment:
Could you add docstrings mentioning this lemma in `continuous_at_cpow`, and conversely? Also, I wonder if `pos_re` or `re_pos` would be better in the lemma name.

Reply:
Done, thanks.